Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM eclipse-temurin:17-jdk-focal
+
+RUN mkdir /app
+
+WORKDIR /app
+
+COPY target/E-CommerceShop-0.0.1-SNAPSHOT.jar /app
+
+EXPOSE 8090
```

Comment:
you've set a different port number than this in your application.properties, make sure to check this